apps/sseramemes/src/image-scripts/addLogoToImage.ts

```diff
@@ -1,20 +1,51 @@
-import Jimp from 'jimp';
+import sharp from 'sharp';
 import * as path from 'path';
 import { resizeImage, TWITTER_IMAGE_SIZE } from './index';
 
-export const addLogoToImage = async (buffer: Buffer): Promise<Buffer> => {
-  const logo: Jimp = await Jimp.read(
-    path.join(process.cwd(), './static/logo.png'),
-  );
-  logo.resize(40, 40);
-  const meme: Jimp = await Jimp.read(buffer);
-  if (
-    meme.getHeight() < TWITTER_IMAGE_SIZE &&
-    meme.getWidth() < TWITTER_IMAGE_SIZE
-  ) {
-    logo.opacity(0.5);
+/**
+ * This is a workaround
+ * @deprecated
+ * @param height
+ * @param width
+ */
+const getLogoWithOpacity = (height: number, width: number): string => {
+  if (height < TWITTER_IMAGE_SIZE && width < TWITTER_IMAGE_SIZE)
```

Comment:
This shouldn't be `||`?

Reply:
I thought the same, but previously for some reason I’ve made it with AND and it is running, do you think I should change to an OR?

Reply:
I think so, sometimes I see a meme where the logo overlaps some text, but didn't find an example

Reply:
Will adjust thx for the feedback